packages/babel-plugin-transform-class-properties/test/fixtures/source-maps/private-get/output.js

```diff
@@ -0,0 +1,15 @@
+var _field = /*#__PURE__*/new WeakMap();
+var _A_brand = /*#__PURE__*/new WeakSet();
+class A {
+  constructor() {
+    babelHelpers.classPrivateMethodInitSpec(this, _A_brand);
+    babelHelpers.classPrivateFieldInitSpec(this, _field, void 0);
+  }
+  method() {
+    babelHelpers.classPrivateFieldGet2(_field, this);
+    babelHelpers.classPrivateGetter(_A_brand, this, _get_getter);
+    babelHelpers.assertClassBrand(_A_brand, this, _method);
```

Comment:
Previously `_field`, `_get_getter` and `_method` where unmapped. Now they are mapped to `#field`, `#getter` and `#method` of the member expression.